next-app/src/components/views/Main/Top/Top.tsx

```diff
@@ -0,0 +1,20 @@
+import * as S from 'components/views/Main/Top/Top.style'
+const Top = () => {
+  return (
+    <S.Wrapper>
+      <S.Title>인사이트가 피둥피둥</S.Title>
+      <S.Subtitle>
+        여기저기 둥둥 떠있는 나의 인사이트 컨텐츠들을 피둥에서 모아보기
+        <br />
+        크롬 새탭에서 바로 시작하세요!
+      </S.Subtitle>
+      <S.GoogleLoginButton>
+        <S.ButtonContentsWrapper onClick={() => {}}>
+          <p>크롬에 추가하기</p>
```

Comment:
```suggestion
         크롬에 추가하기
```
p태그 없이 그냥 버튼명만 써주면 될 것 같아요!

Reply:
수정 완료!!!